docs/content/users/usage/troubleshooting.md

```diff
@@ -118,6 +120,24 @@ nginx   1608 www-data   46u  IPv4  13913      0t0  TCP *:http (LISTEN)
 nginx   5234     root   46u  IPv4  13913      0t0  TCP *:http (LISTEN)
 ```
 
+The `netstat -anv -p tcp` command can also be used:
```

Comment:
Reword to get rid of that impersonal “can also be used” and hopefully describe what the example’s doing before showing it:

```suggestion
You can also use the `netstat -anv -p tcp` command to examine processes running on specific ports:
```